src/api/firebase.js

```diff
@@ -54,7 +54,7 @@ export async function addItem(listId, { itemName, daysUntilNextPurchase }) {
 	const listCollectionRef = collection(db, listId);
 	// TODO: Replace this call to console.log with the appropriate
```

Comment:
**Suggestion/Nit:** You could remove this 'TODO' comment (rows 55-56) since it's been completed.

Reply:
We will delete it!!

Reply:
Done